documentation/docs/practical-sessions/downloading-and-finding-data-from-wis2.md

```diff
@@ -4,6 +4,13 @@ title: Downloading and finding data from WIS2
 
 # Downloading and finding data from WIS2
 
+!!! abstract "Learning outcomes!"
+
+    By the end of this practical session, you will be able to:
+
+    - use pywis-pubsub to subscribe to a Global Broker and download data to your local system
```

Comment:
Capitalise or lower case, both have been used and I've changed one of them to capitals.